values.schema.json

```diff
@@ -197,6 +197,9 @@
                 "resources": {
                     "type": "object"
                 },
+                "hostNetwork": {
+                    "type": "boolean"
+                },
```

Comment:
nit: These are in alphabetical order, so it would be good to move this to be after hmacSecretName.

Reply:
done